tools/test-backend

```diff
@@ -150,6 +158,28 @@ if __name__ == "__main__":
         prof = cProfile.Profile()
         prof.enable()
 
+    #While running --re-run, we access the file containing the 
+    #details of the tests which failed in the last run of test-backend
+    #which is stored at zulip/var/last_test_failure.json and run the failed tests
+    #one by one until the errors are rectified    
+    if options.re_run and options.fatal_errors and not options.force:
```

Comment:
I think we should simplify this conditional.  If we want to prevent folks from using the re_run option with other conflicting arguments, we should enforce this earlier, when we get the options.